composer.json

```diff
@@ -22,15 +22,15 @@
     ],
     "require": {
         "php": "^7.2",
-        "friendsofsymfony/http-cache": "^2.5.2",
+        "friendsofsymfony/http-cache": "^2.6",
```

Comment:
CLEAR does not exit before 2.6: See https://travis-ci.org/github/FriendsOfSymfony/FOSHttpCacheBundle/jobs/714550545

Reply:
yep, its fine to increase the version :+1: